src/libs/actions/PersonalDetails.js

```diff
@@ -262,6 +262,44 @@ function setPersonalDetails(details, shouldGrowl) {
         });
 }
 
+function updateProfile(firstName, lastName, pronouns, timezone) {
+    const myPersonalDetails = personalDetails[currentUserEmail];
+    API.write('UpdateProfile', {
+        details: JSON.stringify({firstName, lastName, pronouns}),
+        timezone: JSON.stringify(timezone),
+    }, {
+        optimisticData: [{
+            onyxMethod: CONST.ONYX.METHOD.MERGE,
+            key: ONYXKEYS.PERSONAL_DETAILS,
+            value: {
+                [currentUserEmail]: {
+                    firstName,
+                    lastName,
+                    pronouns,
+                    timezone,
+                    displayName: getDisplayName(currentUserEmail, {
+                        firstName,
+                        lastName,
+                    }),
```

Comment:
FYI this will need to be added in Web-E, I plan to add this requirement in this issue: https://github.com/Expensify/Expensify/issues/220321

Reply:
Ok, but we should not get any problems when sending this. It will get updated for other users when their refresh, but it is not included in the Pusher update you saying?

Reply:
Right right exactly @mountiny - nothing will break, only the client making the change will get these updates automatically, others will get it after refresh 👍